### ISSUE TITLE
Automatic language detection

### ISSUE THREAD

**Issue by matlapo** [matlapo/BobRoss#2]
Ideally the user shouldn't have to select the language (but should always have the ability to). It should be detected and selected automatically.

**Comment by maximeplante** [comment]
There is this lib which offers highlighting and automatic language detection: https://highlightjs.org

**Comment by matlapo** [comment]
Yeah I've read about it a while ago, thinks for remainding me about it
